config/initializers/redis.rb

```diff
@@ -0,0 +1,3 @@
+def redis
```

Comment:
@jessieay yeah, "Upaya" confuses me too; I chose it because it actually is our app's namespace. My vote, though, goes for putting all application things in _some_ app namespace. Like in Javascript, the best practice of not 'polluting' the global namespace with things.

I'd be totally down for going with `IdentityIdp` if that's the most meaningful name for this app. It definitely is a lot more descriptive than Upaya.